src/resolvers/draftDonationResolver.ts

```diff
@@ -451,4 +462,32 @@ export class DraftDonationResolver {
       return null;
     }
   }
+
+  @Query(_returns => DraftDonation, { nullable: true })
+  async fetchDaftDonationWithUpdatedStatus(
```

Comment:
Pleaser change the name to `fetchDraftDonationWithUpdatedStatus` you have a type error now